### PR TITLE
[15.0][MIG] mrp_flattened_bom_xlsx

### PR DESCRIPTION
**Pull request by JoanMForgeFlow**
Standard migration of the module mrp_flattened_bom_xlsx to version 15.0, depends on: https://github.com/OCA/reporting-engine/pull/554

### CODE REVIEW

**Review by mariadforgeflow — APPROVED** [review]

**Comment by AaronHForgeFlow** [comment]
@sbidoul  could you confirm the test-requirements.txt file is in the correct format here? It seems to me it is, but the installation fails anyways and I am wondering if there is something pending to be updated in the manufacture-reporting repository. Thanks in advance!

**Review by AaronHForgeFlow — COMMENTED** [review]
Functional review fine

**Comment by JoanMForgeFlow** [comment]
> @sbidoul could you confirm the test-requirements.txt file is in the correct format here? It seems to me it is, but the installation fails anyways and I am wondering if there is something pending to be updated in the manufacture-reporting repository. Thanks in advance!

Dependencies solved!

**Review by BernatPForgeFlow — APPROVED** [review]
LGTM!

**Review by LoisRForgeFlow — APPROVED** [review]

**Review by LoisRForgeFlow — COMMENTED** [review]
If this is not really need, remove it, do not comment it

File: mrp_flattened_bom_xlsx/report/flattened_bom_xlsx.xml [hunk 1]
```diff
@@ -11,4 +11,12 @@
...
         attachment_use="False"
     />
+  <!--  <record id="flattened_bom_xlsx" model="ir.actions.report">
+        <field name="name">Export Flattened BOM Structure to Excel</field>
+        <field name="model">mrp.bom</field>
+        <field name="report_type">qweb-pdf</field>
+        <field name="report_name">mrp_flattened_bom_xlsx.flattened_bom_xlsx</field>
+        <field name="report_file">mrp_bom</field>
+        <field name="binding_type">report</field>
+    </record>-->
```
Comment (LoisRForgeFlow) [inline mrp_flattened_bom_xlsx/report/flattened_bom_xlsx.xml]: This

Comment (JoanMForgeFlow) [reply mrp_flattened_bom_xlsx/report/flattened_bom_xlsx.xml]: Done!

**Comment by BernatPForgeFlow** [comment]
@LoisRForgeFlow Could you please merge this PR if everything is alright?

File: mrp_flattened_bom_xlsx/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -1,2 +1,3 @@
-* Héctor Villarreal <hector.villarreal@forgeflow.com>
+* Héctor Vsetsillarreal <hector.villarreal@forgeflow.com>
```
Comment (LoisRForgeFlow) [inline mrp_flattened_bom_xlsx/readme/CONTRIBUTORS.rst]: Could you fix this?

Comment (JoanMForgeFlow) [reply mrp_flattened_bom_xlsx/readme/CONTRIBUTORS.rst]: yes, fixed!

File: mrp_flattened_bom_xlsx/readme/CONTRIBUTORS.rst [hunk 3]
```diff
@@ -1,2 +1,3 @@
-* Héctor Villarreal <hector.villarreal@forgeflow.com>
+* Héctor villarreal <hector.villarreal@forgeflow.com>
```
Comment (LoisRForgeFlow) [inline mrp_flattened_bom_xlsx/readme/CONTRIBUTORS.rst]: I hate to be this nitpicking but... last names start in capital :)

Comment (JoanMForgeFlow) [reply mrp_flattened_bom_xlsx/readme/CONTRIBUTORS.rst]: Done! :+1:

**Review by AaronHForgeFlow — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
:+1:

**Pull request merged**